Parikshit_CHEFADV.cpp

```diff
@@ -0,0 +1,41 @@
+#include<iostream>
```

Comment:
Add links of the question on all

Reply:
My bad, I've taken care in rest all. Thanks.